src/components/marsPhotoViewer/ImageFilter.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useState, ChangeEvent } from "react";
+import "./ImageFilter.scss";
+
+export function ImageFilter() {
+
+    // const currentYear = new Date().getFullYear();
+    // const years = Array.from(new Array(20), (val, index) => currentYear - index);
+    const [selectedRover, setSelectedRover] = useState("Curiosity");
+    const [selectedCamera, setSelectedCamera] = useState("");
+    const [selectedDate, setSelectedDate] = useState("");
```

Comment:
We will need to use these variables in other photoviewer components so need to lift the state up into the parent and pass them into this component as props